### DISCUSSION TITLE
PCL在有打开帮助这个EventType时 是不是会先尝试读取后刷新实际页面

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4763] [提问] [answered]
PCL在有打开帮助这个EventType时
是不是会先尝试读取
测试了个这个（
EventType="打开帮助" EventData="%PID%.json"
其中由于具体的xaml还没开始写（在写索引页
我的EventData使用了一个未存在的帮助json（
![QQ20240919-193345](https://github.com/user-attachments/assets/e7a215b0-157f-4080-a212-243efed74697)
（（（好奇问问 
不是大问题主要是刷新页面就连续给我崩几十个报错出来（（（

**Comment by LTCatt** [comment, marked as answer]
MyListItem 会在加载时立即尝试读取这个文件，来设置标题、图标之类的。
如果不需要这个行为，可以换 MyButton 等。

**Comment by Joker2184** [comment]
哦哦
